devops/cts_exclude_filter_all

```diff
@@ -0,0 +1,2 @@
+# Please use "#" to add comments here.
```

Comment:
this will be used in the future to filter compfails, but is doing nothing today, right?

Reply:
Exactly